src/FordhamBank/Main.java

```diff
@@ -86,11 +77,26 @@ public void start(Stage primaryStage) {
         root.getChildren().add(main);
 
         Scene scene = new Scene(root, 800, 600);
+       
+        
         
         primaryStage.setTitle("Accounts Summary");
         primaryStage.setScene(scene);
         primaryStage.show();
     }
+    
+    private void setAccounts(User user) {
+    	BankAccount account1 = new BankAccount(user.GetId(), AccountType.CHECKING, "My Checking");
```

Comment:
Add transactions to each BankAccount in this method instead of creating them within the `Transaction` class. The way you wrote it all the `BankAccounts` have the same Transactions to it is hard to test and not as useful. For each `BankAccount` created here, call `AddTransaction` and add atleast 3 different transactions for each account.